packages/appium/lib/cli/parser.js

```diff
@@ -144,6 +148,28 @@ class ArgParser {
     }
   }
 
+  /**
+   * Fix values of parsed arguments.
+   *
+   * @param {object} args
+   * @returns {object}
+   */
+  static _fixArgValueFormat(args) {
+
+    // "--log-filters" argument parsed as an string instead of JSON object.
+    // Current definition in appium-config-scheme.js works expectedly for
+    // appium config. We may be able to update the scheme properly to
+    // make it work for both, "--log-filters" and "--config", but
+    // it may take more time.
+    // FIXME: Possibly 'appiumCliTransformer' in scheme does not work properly(?)
```

Comment:
cannot we patch it directly in appiumCliTransformer?

Reply:
For now, I guess we could do something in https://github.com/appium/appium/blob/master/packages/appium/lib/schema/cli-args.js#L99, but for this usage, I'm still not able to find a better place than here. I thought something appium-config-schema.js related, but possibly my assumption was wrong for this FIXME..